File: Python/interactions/infinite_scroll.py
from playwright.sync_api import sync_playwright

with sync_playwright() as p:
    browser = p.chromium.launch()
    page = browser.new_page()
    page.goto("https://example.com/infinite-scroll")
    for _ in range(10):
        page.evaluate("window.scrollBy(0, window.innerHeight);")
        page.wait_for_timeout(1000)
    print(page.content())
    browser.close()

File: NodeJS/interactions/infinite_scroll.js
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();
  await page.goto('https://example.com/infinite-scroll');
  for (let i = 0; i < 10; i++) {
    await page.evaluate(() => window.scrollBy(0, window.innerHeight));
    await page.waitForTimeout(1000);
  }
  console.log(await page.content());
  await browser.close();
})();
